Userland/Applications/ThemeEditor/main.cpp

```diff
@@ -374,7 +374,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             theme->write_entry("Paths", to_string(role), preview_widget.preview_palette().path(role));
         }
 
-        theme->sync();
+        if (theme->sync().is_error()) {
+            // FIXME: Expose this to the user, since failing to save is important to know about!
+            dbgln("Failed to save theme file.");
```

Comment:
Can we log the error itself as well? I think `Error` has a formatter, so just:
```c++
if (auto result = theme->sync(); result.is_error()) {
    // FIXME: Expose this to the user, since failing to save is important to know about!
    dbgln("Failed to save theme file: {}", result.error());
}
```

Reply:
And same comment on the `dbgln`s below this :)

Reply:
Thanks! You're right. :^) I think I got all of them.